src/VibrosonicsAPI.cpp

```diff
@@ -21,34 +21,44 @@ void VibrosonicsAPI::init()
  */
 void VibrosonicsAPI::processAudioInput(float output[])
 {
+    // Copy samples from rolling buffer to FFT input array
+    for (int i = 0; i < WINDOW_SIZE_OVERLAP; i++) {
+      vData[i] = rollingInputBuffer[i];
+    }
```

Comment:
Minor Suggestion: A for loop over a memcpy or std::copy may affect output latency. We can try using the following line to hopefully speed up the function,

NOTE: #include <algorithm> would be needed at the top of the file.
```suggestion
    std::copy(rollingInputBuffer, rollingInputBuffer + WINDOW_SIZE_OVERLAP, vData);
```